src/app/pages/Projects.tsx

```diff
@@ -87,10 +89,34 @@ function Projects() {
       const reader = new FileReader();
       reader.readAsText(acceptedFiles[0]);
       reader.onload = function() {
-        setOntology(reader.result as string);        
+        setOntology(reader.result as string);  
+        const defaultConfig = createDefaultConfig(reader.result as string); 
+        setConfig(JSON.stringify(defaultConfig, null, 2));    
       }
     }
-  }  
+  }
+  
+  function createDefaultConfig(ontology: string) {
+    const ontologyJson = JSON.parse(ontology);
+    let config : IProjectConfigs = {legendConfigs: {}, scopeConfigs: {}};
+    for (const entry of ontologyJson.definitions) {
+      const entryName = entry.entry_name;
+      config['scopeConfigs'][entryName] = false;
```

Comment:
Only subtypes of "forte.data.annotation" can be used as scope, because these types contain "begin", "end".

Reply:
Done.